src/index.js

```diff
@@ -1,62 +1,108 @@
-  heading.innerHTML = '<p>Today\'s To Do</p> <i class="fas fa-sync-alt icons"></i>';
-  list.append(heading);
-
-  const input = document.createElement('li');
-  input.innerHTML = '<input class="text" type="text" placeholder="Add to your list..."> <i class="fas fa-stream icons"></i>';
-  list.append(input);
-
-  for (let i = 0; i < toDoList.length; i += 1) {
-    if (toDoList[0].index === i) {
-      const listCode = `<li data-id="${i}"><div id="${i}" class="task"><input class="check-box list-${i}" type="checkbox"><p>${toDoList[i].description}</p></div>
-        <i class="fas fa-ellipsis-v icons"></i>`;
-      list.innerHTML += listCode;
-    } else if (toDoList[1].index === i) {
-      const listCode = `<li data-id="${i}"><div id="${i}" class="task"><input class="check-box list-${i}" type="checkbox"><p>${toDoList[i].description}</p></div>
-        <i class="fas fa-ellipsis-v icons"></i>`;
-      list.innerHTML += listCode;
-    }
-  }
+  const listsCode = toDoList.map((newList) => new ToDoClass(
+    newList.description,
+    newList.completed,
+    newList.index,
+  ).listCode());
+
+  list.innerHTML = listsCode.join('');
+
+  const checkbox = document.querySelectorAll('.check-box');
+  checkBox(checkbox);
 
-  const btn = document.createElement('li');
-  btn.innerHTML = '<button type="button">Clear all completed</button>';
-  list.append(btn);
+  const btn = document.querySelectorAll('.icon-d');
+  btn.forEach((el) => {
+    el.addEventListener('click', (e) => {
+      const oldList = e.target.parentElement.parentElement.getAttribute('data-id'); // get parent
+      ToDoClass.remove(oldList);
+      showList();
+    });
+  });
 }
 
 showList();
 
 document.addEventListener('DOMContentLoaded', () => {
-  if (toDoList[0].completed === true) {
-    document.getElementById('0').classList.add('strike');
-    document.querySelector('.list-0').checked = true;
-  }
+  const submitBtn = document.querySelector('.sub-icon');
+  submitBtn.addEventListener('click', () => {
+    const desInput = document.querySelector('.text');
+    const description = desInput.value.trim();
+    if (!description) {
+      return;
+    }
 
-  if (toDoList[1].completed === true) {
-    document.getElementById('1').classList.add('strike');
-    document.querySelector('.list-1').checked = true;
-  }
+    const completed = false;
+    const newList = new ToDoClass(description, completed);
+    ToDoClass.addTo(newList);
+    showList();
+    desInput.value = '';
+  });
 
-  const checkbox = document.querySelectorAll('.check-box');
-  checkBox(checkbox);
+  const clearCompleted = document.querySelector('.clear-all');
+  clearCompleted.addEventListener('click', () => {
+    toDoList = toDoList.filter((x) => x.completed === false);
+    showList();
+    for (let i = 0; i < toDoList.length; i += 1) {
+      toDoList[i].index = i;
+    }
+    localStorage.setItem('ToDo', JSON.stringify(toDoList));
+  });
+
+  const isComp = document.querySelectorAll('.check-box');
```

Comment:
- [ ] Please name your variables in a way that they reveal the intention behind them. This way they become searchable and easier to understand after a person sees them. In this case, it would be better to name this variable `isComplete` for example.